homeassistant/components/ambient_network/const.py

```diff
@@ -0,0 +1,22 @@
+"""Constants for the Ambient Weather Network integration."""
+
+from datetime import timedelta
+import logging
+
+DOMAIN = "ambient_network"
+
+ENTITY_MAC_ADDRESS = "mac_address"
+ENTITY_STATION_NAME = "station_name"
```

Comment:
For stuff in the config entry data or options we rather use `CONF_`

Reply:
Renamed.